CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 <!--
 Contains editorialized release notes. Raw release notes should go into `RELEASE-NOTES.txt`.
 -->
+## 11.2
+For this release, we focused on polishing a few things and taking care of minor bugs that our team found, including the option to preview your products before publishing them. Don’t be fooled – we are working on some pretty cool stuff for you all! Keep sharing feedback 💜
```

Comment:
@anitaa1990 I just wanted to let you know that App Store Connect didn't allow the 💜 character and I had to replace it with `!` in https://github.com/woocommerce/woocommerce-ios/pull/8153 before submitting `11.2`.